contract-addresses/mainnetAddresses.ts

```diff
@@ -97,9 +97,9 @@ const MainnetAddresses = {
     artifactName: 'ERC20CompoundPCVDeposit',
     address: '0x243C601CC5DaA3Ac250B14509804188347bd2aFB'
   },
-  curve3crv: { artifactName: 'IStableSwap3', address: '0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490' },
-  curve3pool: { artifactName: 'IStableSwap3', address: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7' },
-  curveMetapool: { artifactName: 'IStableSwap2', address: '0x06cb22615ba53e60d67bf6c341a0fd5e718e1655' },
+  curve3crv: { artifactName: 'unknown', address: '0x6c3F90f043a72FA612cbac8115EE7e52BDe6E490' },
+  curve3pool: { artifactName: 'unknown', address: '0xbebc44782c7db0a1a60cb6fe97d0b483032ff1c7' },
+  curveMetapool: { artifactName: 'unknown', address: '0x06cb22615ba53e60d67bf6c341a0fd5e718e1655' },
```

Comment:
Why change the variable names and artifacts?

Reply:
@ElliotFriedman the "IStableSwap" contracts aren't in the repo.
When the loader tries to grab the artifact and throw this into the mainnetContracts, it throws an error, causing an integration test to fail.